{{ cookiecutter.package_name }}/.readthedocs.yaml

```diff
@@ -0,0 +1,31 @@
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "mambaforge-4.10"
+  jobs:
+    post_checkout:
+      - git fetch --unshallow || true
+    pre_install:
+      - git update-index --assume-unchanged .rtd-environment.yml docs/conf.py
+
+conda:
+  environment: .rtd-environment.yml
+
+sphinx:
+  builder: html
+  configuration: docs/conf.py
+  fail_on_warning: false
+
+formats:
+  - htmlzip
+
+python:
+  install:
+    - method: pip
+      extra_requirements:
+        - all
+        - docs
+        - docs-gallery
```

Comment:
Did a separate gallery build get added to the template?

Reply:
No, we haven't galleried yet